### PR TITLE
Add ChainGovernance.sol and chain_governance.go

### PR DESCRIPTION
**Pull request by sulenn**

### CODE REVIEW

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/utils.go  5
- precompiled/chaingovernance/chain_governance.go  2
- precompiled/chaingovernance/chain_governance_test.go  5
- precompiled/chaingovernance/chain_governance_service.go  4
- precompiled/config/system_config_service.go  2
         

Clones added
============
- precompiled/chaingovernance/chain_governance.go  5
- precompiled/chaingovernance/chain_governance_test.go  3
- precompiled/consensus/consensus_service_test.go  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5862800&bid=19192763)

File: core/types/log.go [hunk 1]
```diff
@@ -55,12 +54,12 @@ type Log struct {
 	Removed bool `json:"removed"`
 }
 
-type logMarshaling struct {
-	Data        hexutil.Bytes
-	BlockNumber hexutil.Uint64
-	TxIndex     hexutil.Uint
-	Index       hexutil.Uint
-}
+// type logMarshaling struct {
```
Comment (bxq2011hust) [inline core/types/log.go:57]: why？

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/chaingovernance/chain_governance.go  2
- precompiled/config/system_config_service.go  2
- precompiled/chaingovernance/chain_governance_test.go  5
- precompiled/chaingovernance/chain_governance_service.go  4
- precompiled/utils.go  5
         

Clones added
============
- precompiled/chaingovernance/chain_governance.go  5
- precompiled/chaingovernance/chain_governance_test.go  3
- precompiled/consensus/consensus_service_test.go  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5862800&bid=19192763)

File: core/types/log.go (hunk 1 above)
Comment (sulenn) [reply core/types/log.go:57]: because it is not used, checks by code style.

File: precompiled/utils.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	"math"
+	"math/big"
+	"strings"
+
+	"github.com/FISCO-BCOS/go-sdk/core/types"
+
+	"github.com/FISCO-BCOS/go-sdk/abi"
+)
+
+func GetPreServiceOutput(abiStr, name string, receipt *types.Receipt) (int, error) {
```
Comment (bxq2011hust) [inline precompiled/utils.go:15]:
```suggestion
func ParseInt64Output(abiStr, name string, receipt *types.Receipt) (int, error) {
```

**Comment by fisco-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- precompiled/chaingovernance/chain_governance.go  2
         

Clones added
============
- precompiled/chaingovernance/chain_governance.go  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5862800&bid=19192763)

**Review by bxq2011hust — APPROVED** [review]

**Pull request merged**